src/WS.cpp

```diff
@@ -207,12 +234,15 @@ enum
     PNT_STREAM_ROTATION,
     PNT_STREAM_SCALE_WIDTH,
     PNT_STREAM_SCALE_HEIGHT,
+    PNT_STREAM_STATS,
     PNT_STREAM_OSD
 };
 
 static const char *const stream_keys[] = {
```

Comment:
Are there any equivalent keys we need to update for web sockets? i.e. websocket.usertoken

Reply:
For security reasons I have not implemented websocket configuration customizations via websocket or html.